frontend/www/js/omegaup/graderv2/IDESettings.vue

```diff
@@ -0,0 +1,227 @@
+          max="5.0"
+          min="0.1"
+          step="0.1"
+          type="number"
+        />
+      </div>
+      <div class="form-group col-md-4">
+        <label for="inputOverallWallTimeLimit">{{
+          T.settingsOverallWallTimeLimit
+        }}</label>
+        <input
+          v-model="overallWallTimeLimit"
+          class="form-control"
+          max="5.0"
+          min="0.1"
+          step="0.1"
+          type="number"
+        />
+      </div>
+      <div class="form-group col-md-4">
+        <label for="inputExtraWallTime">{{ T.settingsExtraWallTime }}</label>
+        <input
+          v-model="extraWallTime"
+          class="form-control"
+          max="5.0"
+          min="0.0"
+          step="0.1"
+          type="number"
+        />
+      </div>
+    </div>
+    <div class="form-row">
+      <div class="form-group col-md-6">
+        <label for="inputMemoryLimit">{{ T.settingsMemoryLimit }}</label>
+        <input
+          v-model="memoryLimit"
+          class="form-control"
+          max="1073741824"
+          min="33554432"
+          step="1048576"
+          type="number"
+        />
+      </div>
+      <div class="form-group col-md-6">
+        <label for="inputOutputLimit">{{ T.settingsOutputLimit }}</label>
+        <input
+          v-model="outputLimit"
+          class="form-control"
+          max="104857600"
+          min="0"
+          step="1024"
+          type="number"
+        />
+      </div>
+    </div>
+    <div class="form-row">
+      <div class="form-group col-md-6">
+        <label for="inputValidator">{{ T.settingsValidator }}</label>
+        <select v-model="validator" class="form-control">
+          <option value="custom">Custom</option>
+          <option value="literal">Literal</option>
+          <option value="token">Token</option>
+          <option value="token-caseless">Token (Caseless)</option>
+          <option value="token-numeric">Token (Numeric)</option>
+        </select>
+      </div>
+      <div v-if="validator == 'token-numeric'" class="form-group col-md-6">
+        <label for="inputTolerance">{{ T.settingsTolerance }}</label>
+        <input
+          v-model="tolerance"
+          class="form-control"
+          max="1"
+          min="0"
+          type="number"
+        />
+      </div>
+      <div v-if="validator == 'custom'" class="form-group col-md-6">
+        <label for="inputValidatorLanguage">{{ T.settingsLanguage }}</label>
+        <select v-model="validatorLanguage" class="form-control">
+          <option value="cpp17-gcc">C++17</option>
+          <option value="py3">Python 3.6</option>
+        </select>
+      </div>
+    </div>
+    <div class="form-row">
+      <div class="form-group col-md-4">
+        <label for="inputInteractive">{{ T.settingsInteractive }}</label>
+        <omegaup-radio-swtich
+          :value.sync="interactive"
+          :selected-value="interactive"
+          :name="'interactive'"
```

Comment:
If this attribute will always be 'interactive', it doesn't need reactivity, so:
```suggestion
          name="interactive"
```
is fine. 

Or even better, if you are not sending this form via POST, probably you don't need to assign it this attribute